docs/v1/api_bindings.md

```diff
@@ -47,6 +47,121 @@ curl -X POST -H "Content-Type: application/json"
  }
   ```
   
-  ### Errors
+### Errors
```

Comment:
Please add at the beginning what is a binding, host and all that are used int bindings

Reply:
Added a brief description of the binding entity at the top of the api_bindings.md file